ADAL/automation/tests/interactive/ADALGuestUsersLoginTest.m

```diff
@@ -127,7 +129,8 @@ - (void)testInteractiveAndSilentAADLogin_withPromptAlways_noLoginHint_ADALWebVie
     guestParams = [self.testConfiguration configWithAdditionalConfiguration:guestParams account:self.primaryAccount];
     [self acquireToken:guestParams];
     [self aadEnterEmail];
-    [self adfsEnterPassword];
+    [self guestEnterUsername];
```

Comment:
Means in the ADFS, we need to enter the username again?

Reply:
Lab changed guest user configuration and it now returns ADFS setup, where you need to enter your username again...